llvm/test/tools/llvm-lib/sym64-threshold.test

```diff
@@ -0,0 +1,68 @@
+# RUN: yaml2obj --docnum=1 %s -o %t01234567890234567789.obj
+# RUN: yaml2obj --docnum=2 %s -o %t-ec.obj
+# RUN: env SYM64_THRESHOLD=100 llvm-lib -machine:amd64 -out:%t.lib %t01234567890234567789.obj
+# RUN: llvm-readobj %t.lib
```

Comment:
Can we check some aspect of the output here, instead of just requiring that `llvm-readobj` exits successfully?

Reply:
I changed it to check archive map with `llvm-nm` instead.